module/rdpClientCon.c

```diff
@@ -1127,7 +1127,7 @@ rdpClientConInit(rdpPtr dev)
                 return 0;
             }
         }
-        g_chmod_hex("/tmp/.xrdp", 0x1777);
+        g_chmod_hex("/tmp/.xrdp", 0x3777);
```

Comment:
Anything that touches suid and sgid bits should be documented. Also, it's it highly desirable that such behavior can be configured using configuration files. Creating a directory with a predictable name in a world writable directory was bad already. Creating a sgid directory makes it even worse. What if an attacker replaces /tmp/.xrdp with a link to an executable and exploits this code to get sgid permissions? What if the process group is root or wheel?